packages/core/getUserAvailability.ts

```diff
@@ -520,10 +521,43 @@ const _getBusyTimesFromLimits = async (
     performance.measure(`checking duration limits took $1'`, "durationLimitsStart", "durationLimitsEnd");
   }
 
+  const busyTimes = limitManager.getBusyTimes().sort((a, b) => (dayjs(a.start).isBefore(b.start) ? -1 : 1));
+
+  if (eventType.seatsPerTimeSlot) {
+    for (let i = 0; i < bookings.length; i++) {
+      const booking = bookings[i];
+
+      if (
+        booking.attendeesCount &&
+        booking.attendeesCount < eventType.seatsPerTimeSlot &&
+        dayjs(booking.start).isBetween(dateFrom, dateTo)
+      ) {
+        for (let j = i; j < busyTimes.length; j++) {
```

Comment:
To fix https://github.com/calcom/cal.com/pull/13526#pullrequestreview-2016014929 
```suggestion
        for (let j = 0; j < busyTimes.length; j++) {
```
`let j = i` will only work if the bookings are sorted.